spec/json/twilio_preview_bulk_exports.json

```diff
@@ -0,0 +1,710 @@
+{
```

Comment:
I don't think these should have been split up

Reply:
Yeah the reason they are split up is because the api-definitions structure is different just for preview where there is no v1/v2 (https://code.hq.twilio.com/twilio/api-definitions/tree/master/base/preview). Should the logic be something special for preview or should we force the structure to conform with the other products? The last time I brought this up, I remember Sam mentioning that preview isn't really used and the cli code actually ignores preview. Should we just ignore the spec generation for preview?